.circleci/config.yml

```diff
@@ -106,13 +96,30 @@ jobs:
             .coverage.unittest-27
 
   unittest-26:
+    working_directory: ~/scalyr-agent-2
     docker:
-      - image: circleci/python:2.7-jessie
+      - image: dockershelf/python:2.6-dev
+        environment:
+          PYTHONPATH=./scalyr_agent/third_party
```

Comment:
Since scalyr_init modifies the path, do you need this?  (Just going based on your other comments)

Reply:
You are correct, I will remove it.

Reply:
Removed in e15c2eed6d488a938a6954322b733d225aa51b30.